src/modules/dashboard/components/Extensions/ExtensionSetup.tsx

```diff
@@ -58,10 +65,29 @@ const ExtensionSetup = ({
     history.replace(`/colony/${colonyName}/extensions/${extensionId}`);
   }, [history, colonyName, extensionId]);
 
-  const transform = useCallback(mergePayload({ colonyAddress, extensionId }), [
-    colonyAddress,
-    extensionId,
-  ]);
+  const transform = useCallback(
+    pipe(
+      mapPayload((payload) => {
+        if (extensionId === Extension.VotingReputation) {
+          const formattedPayload = {};
+          initializationParams?.map(({ paramName }) => {
+            if (endsWith(paramName, 'Period')) {
+              formattedPayload[paramName] = payload[paramName] * 3600; // Seconds in 1 hour
+            } else {
+              formattedPayload[paramName] = bigNumberify(
+                payload[paramName],
+              ).mul(bigNumberify(10).pow(16));
```

Comment:
Yep, this is how Alex explained it to me ,as I had errors pop up when I tried to use `18`:

```
it's because you're dealing with a percentage
For these fixed-point values, 1 is represented as 10**18 on chain
But 100% also represents 1
To go from a percentage to a fraction, you divide by 100, and then to go to the fixed-point representation, you multiply by 10**18
Which is a net effect of multiplying by 10**16
```